### ISSUE TITLE
Feature Request - DualQuickSlots

### ISSUE THREAD

**Issue by Ikepuska** [Godofdrakes/WeaponPalettes#6]
I'm not sure if the issue is with your mod, or [DualQuickSlots](https://github.com/Faeryn/Outward_DualQuickSlots), but when you switch between quickslots in weaponpalettes it clears the offhand weapon, so when you switch back it's gone.

It'd be great if the two mods would be compatible so that they could be used together.

**Comment by Godofdrakes** [comment]
Hmmmm. Might be possible. It looks like instead of creating a save data extension they add to the string the game itself serializes. If I serialized/deserialized that data instead of the item id it would, in theory, preserve any data other mods append to it.

**Comment by Faeryn** [comment]
Yep! Originally I was using save data extension (loading from it is still there, for backwards compatibility) but then I had to make the mod compatible with Multiple Quick Slot Bars, so I figured it's the easiest way. If people start adding stuff to quickslot data it may worth just replacing it with a json or something, I'm not a fan of this vanilla "csv-like" way of serializing stuff.
